clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/events/DeleteServerGroupEvent.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.orchestration.events;
+
+public class DeleteServerGroupEvent implements OperationEvent {
+  private final OperationEvent.Type type = OperationEvent.Type.SERVER_GROUP;
+  private final OperationEvent.Action action = OperationEvent.Action.DELETE;
+
+  private final String cloudProvider;
+
+  private final String accountId;
+  private final String region;
```

Comment:
worth checking whether this is a good enough assumption on the set of coordinates that identify a server group across all providers (thinking of the GCE zone/region shenanigoats that are all over orca)?